src/views/BaseView.vue

```diff
@@ -1,42 +1,51 @@
 <template>
-  <v-layout>
-    <!--          :clipped="$vuetify.breakpoint.lgAndUp"-->
+  <v-app>
     <v-navigation-drawer
-      color="sidebar"
-      :expand-on-hover="true"
+      v-model="drawer"
       permanent
       rail
-      role="navigation"
+      expand-on-hover
+      color="#378dc5"
       theme="dark"
+      role="navigation"
+      class="sidebar-nav"
     >
-      <v-list-item
-        v-for="(item, index) in navItems"
-        :id="`sidebar-link-${kebabCase(item.title)}`"
-        :key="`sidebar-link-${index}`"
-        :aria-current="route.path === item.path"
-        class="nav-list-item"
-        link
-        tag="a"
-        @click="router.push(item.path)"
-      >
-        <template #prepend>
-          <v-icon color="icon-nav-default" :icon="item.icon" />
+      <v-list>
+        <template v-for="(item, i) in navItems" :key="i">
+          <v-list-item
```

Comment:
@mohamalnadi I believe we need to keep the `aria-current` and `tag="a"` on the list items for accessibility, unless you verified that they were still present in the rendered HTML after your changes?